BothamNetworking/BothamAPIClient.swift

```diff
@@ -134,4 +143,8 @@ public class BothamAPIClient {
         return httpClient.hasValidScheme(request)
 
     }
+}
+
+private struct BothamAPIClientErrorCode {
+    private static let networkError = -1009
```

Comment:
It look there is a bunch of others error code.
from -998 to -1021
Maybe we should we go the other way wrap error if it not returing a valid http status code.